test/main.sh

```diff
@@ -635,6 +636,7 @@ if [ "${1:-"all"}" != "cluster" ] && [ "${1:-"all"}" != "standalone" ]; then
     run_test test_snap_vm_empty "snap empty VM"
     run_test test_snap_lxd_user "snap lxd-user"
     run_test test_snap_storage_volume_attach_vm "snap attaching storage volumes to VMs"
+    run_test test_snap_basic_usage_container "snap basic usage container"
```

Comment:
Things that are not snap specific should be kept as regular (non-snap) tests. I recently dropped such basic usage snap test because of that.